platform/broadcom/saibcm-modules/make/Make.config

```diff
@@ -173,6 +173,9 @@ CXXFLAGS += ${INCFLAGS}
 CPPFLAGS += ${INCFLAGS}
 
 CFLAGS += -DSAI_FIXUP -UKCOM_FILTER_MAX -DKCOM_FILTER_MAX=1024
+ifeq ($(ASIC_FAMILY),dnx)
```

Comment:
is this upstreamed? does broadcom agree with such macro?

Reply:
Broadcom doesn't need this in their SAI build environment as they already use the **SAI_PLATFORM** in their Makefile and add the CFLAGS as below. 
```
ifeq ($(SAI_PLATFORM),x86-dnx-deb)
CFLAGS += -DBCM_SAND_SUPPORT -DBCM_DNX_SUPPORT -DBCM_DNX2_SUPPORT -DBCM_DNXF_SUPPORT
endif
```
I did a similar approach with ASIC_FAMILY env parameter here in saibcm-modules.

Reply:
With the new commit SAI 5.0.0.3, Broadcom has started to use SDK 6.5.22 for DNX platforms. In discussion with broadcom whether we need to create separate saibcm-modules-dnx with this new 6.5.22 GPL module or not. 

The XGS platforms still continue using 6.5.21 SDK/GPL.